sdk/core/azure-core/test/ut/curl_connection_pool.cpp

```diff
@@ -36,8 +36,8 @@ namespace Azure { namespace Core { namespace Test {
 
       // Use the same request for all connections.
       Azure::Core::Http::Request req(
-          Azure::Core::Http::HttpMethod::Get, Azure::Core::Url("http://httpbin.org/get"));
-      std::string const expectedConnectionKey = "httpbin.org0011";
+          Azure::Core::Http::HttpMethod::Get, Azure::Core::Url(AzureSdkHttpbinServer::Get()));
+      std::string const expectedConnectionKey = "azure-sdk-for-cpp-httpbin.azurewebsites.net0011";
```

Comment:
What subscription is this in? If I found the correct place it looks like it is currently in our playground subscription which is probably not the correct place for it. We should probably put this in our test subscription. 

I don't know a lot about httpbin but it seems like there is some overlap with some test proxy work that @scbedd is working on, so it might be worth a chat between you guys.